tests/helpers/non_sklearn_metrics.py

```diff
@@ -0,0 +1,52 @@
+"""File for non sklearn metrics that are to be used for reference for tests"""
+
+import numpy as np
+from sklearn.metrics._regression import _check_reg_targets
+from sklearn.utils.validation import check_consistent_length
+
+
+def symmetric_mean_absolute_percentage_error(y_true, y_pred, sample_weight=None, multioutput='uniform_average'):
+    """Symmetric mean absolute percentage error regression loss.
```

Comment:
could you insert a reference here just to be sure which definition you used, if we need to debug in the future

Reply:
I've added the wiki page and the formula for reference